demos/browser/server.js

```diff
@@ -22,14 +23,16 @@ const indexPage = fs.readFileSync(indexPagePath);
 // Create ans AWS SDK Chime object. Region 'us-east-1' is currently required.
 // Use the MediaRegion property below in CreateMeeting to select the region
 // the meeting is hosted in.
-const chime = new AWS.Chime({ region: 'us-east-1' });
-const sts = new AWS.STS({ region: 'us-east-1' })
+const currentRegion = 'us-east-1';
+const sts = new AWS.STS({ region: currentRegion });
 
 // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com.
 const endpoint = process.env.ENDPOINT || 'https://service.chime.aws.amazon.com';
```

Comment:
Have we tested with gamma endpoint and non-us-east-1region passed in? Does it work?

Reply:
no i have not tested the gamma endpoint but i can deploy and test.

Reply:
Updated the demo to be configurable for gamma endpoints and tested it by deploying it on gamma